README.rst

```diff
@@ -73,3 +73,13 @@ Utility functions that don't fit elsewhere.
 ~~~~~~~~~~~~~~
 
 Operations available from the `guild.ai <https://guild.ai/>`__ CLI.
+
+Testing
```

Comment:
Testing and developing locally?

Reply:
This is only about testing tho. You read that section if you're developing locally and then you're like "ooh, I should test this"